zipline/utils/test_utils.py

```diff
@@ -881,3 +881,9 @@ def decorator(f):
         param_sets = product(*(params[name] for name in argnames))
         return subtest(param_sets, *argnames)(f)
     return decorator
+
+
+def make_test_handler(testcase, *args, **kwargs):
+    handler = TestHandler(*args, **kwargs)
+    testcase.addCleanup(handler.close)
```

Comment:
err, I think that just means it is after tearDown and not in tearDownClass. This is the behavior we want though so it is okay